web/src/components/navbar/NavBar.test.tsx

```diff
@@ -172,36 +195,43 @@ describe("<NavBar />", () => {
       expect(screen.queryByText(Config.navigationDefaults.logInButton)).not.toBeInTheDocument();
     });
 
+    it("displays myNJ button when user is authenticated", async () => {
+      useMockUserData({});
+      renderDesktopNav();
+      expect(screen.queryByText(Config.navigationDefaults.profileLinkText)).not.toBeInTheDocument();
+    });
+
+    it("displays log out button when user is authenticated", async () => {
+      useMockUserData({});
+      renderDesktopNav();
+      expect(screen.queryByText(Config.navigationDefaults.logInButton)).not.toBeInTheDocument();
+    });
+
     it("displays a closed dropdown menu on the NavBar", () => {
-      const user = { name: "John Smith", email: "test@example.com" };
+      const userData = generateBusinessNamedUserData();
+      useMockUserData({ ...userData });
 
-      useMockUserData({
-        user: generateUser(user),
-      });
       renderDesktopNav();
-      const menuEl = screen.getByText(user.name);
+      const menuEl = screen.getByText(getNavBarBusinessTitle(userData));
```

Comment:
I would generally recommend against using a helper function like this in tests. Shouldn't we know what the name would be in the context of the test data? we can just select by the text directly

Reply:
Yeah, we can manually figure out what the name should be and look for that